File: C_Program/average.c
//wap to accept 3 nos and print total and average

#include<stdio.h>
void main() {
	int n1,n2,n3,total;
	float average;
	printf("enter the 3 nos: ");
	scanf("%d%d%d",&n1,&n2,&n3);
	total=n1+n2+n3;
	printf("\n total=%d",total);
	average=total/3;
	printf("\n average=%.2f",average);
}

File: Java_Programs/Average.java
//wap to accept 3 nos and print total and average.
import java.util.*;
class Average
{
	public static void main(String []args)
	{
		int n1,n2,n3;

		Scanner s = new Scanner(System.in);

		System.out.println("Enter n1 value: ");
		n1 = s.nextInt();
		System.out.println("Enetr n2 value: ");
		n2 = s.nextInt();
		System.out.println("Enter n3 value: ");
		n3 = s.nextInt();

		int total = n1+n2+n3;
		System.out.println("Total = " + total);

		float average = (float)(total/3);
		System.out.println("Average = " + average);
	}
}